vault/config/overlays/nerc-ocp-infra/config/nerc-ocp-beta-test.yaml

```
@@ -0,0 +1,26 @@
+auth:
+- type: kubernetes
+  path: kubernetes/hypershift1
```

You need to fix the path so that it has the appropriate cluster name:

```suggestion
  path: kubernetes/nerc-ocp-beta-test
```